pkl-cli/src/main/kotlin/org/pkl/cli/CliEvaluator.kt

```diff
@@ -158,7 +160,7 @@ constructor(
           } else {
             if (output.isNotEmpty()) {
               outputFile.writeString(
-                options.moduleOutputSeparator + IoUtils.getLineSeparator(),
+                options.moduleOutputSeparator + '\n',
```

Comment:
Changed this to be `LF` and not platform dependent. Otherwise, `pkl eval foo.pkl bar.pkl` will give you different results on different OSes, which seems quite undesirable.

Reply:
This is configuration output meant to be saved to a file, or applied to a system.

The standard use-case here is to generate a YAML document stream. For example, `pkl eval podA.pkl podB.pkl > output.yaml`.

Note that if the `-o` option is set, it already uses `LF` endings. So, without this change, these two would have had different results:

```
pkl eval podA.pkl podB.pkl > output.yaml
```

```
pkl eval podA.pkl podB.pkl -o output.yaml
```